contact.html

```diff
@@ -22,23 +24,24 @@
 		</div>
 		<div class="contactdetails">
```

Comment:
When you tab through on this page, it looks like your.. padding? or margin? is slightly off on the buttons because the right side of the focus is not visible. Very minor thing though.